scripts/printProcessQueue.py

```diff
@@ -76,6 +87,26 @@ def output_text(items):
 
     dbu = DButils.DButils(options.database)
     items = dbu.ProcessqueueGetAll()
+     
+    # Check if the processque is empty 
+    # Usually used for bash scripts
+    #   return 0 if data exists (true)
+    #   return 1 if processque is empty (false)
+    if( options.exist ):
+        if( len(items) != 0 ): 
+            print("Process Queue is not empty\n" if not options.quiet else "", end='')
+            del dbu
+            sys.exit(0)
+        else:
+            del dbu
+            sys.exit(1)
+
+    # return the number of items in processqueue 
+    if( options.count ):
```

Comment:
The if statements shouldn't have parentheses (using PEP8 as an example). Also `if( len(items) != 0 )` can just be `if items`.

Reply:
If items sounds better here.

Reply:
Ah, I see we had some of those weird parens in the script already, that makes more sense...fixed them as well.